_episodes/09-remotes.md

```diff
@@ -141,25 +154,25 @@ We now want to try the second option that GitHub suggests:
 
 1. Now go to your guacamole repository on your computer.
 2. Check that you are in the right place with `git status`.
-3. Copy paste the two lines to the terminal and execute those, in my case (**you
+3. Copy paste the three lines to the terminal and execute those, in my case (**you
   need to replace the "user" part and possibly also the repository name**):
 
 ```shell
-$ git remote add origin https://github.com/user/recipe.git
-$ git push -u origin master
+$ git remote add origin https://github.com/user/recipe.git
+$ git branch -M main
```

Comment:
This, by default, renames the current branch without an `<oldbranch>` argument?  The manual page doesn't specify, and it's a news to me you don't need that!

Can we say what these three lines mean?

Reply:
I didn't know this flag until today.